### PR TITLE
Event API mapping

### PR DESCRIPTION
**Pull request by felipecabargas**
Adds wrapper functions around available Event API methods tp call them inside `monsido_app`

### CODE REVIEW

File: lib/piwik/events.rb [hunk 1]
```diff
@@ -12,4 +12,40 @@ class Events < ApiModule
       getCategoryFromNameId
     }
   end
-end
\ No newline at end of file
+
+  def category params = {}
```
Comment (serch) [inline lib/piwik/events.rb:16]: any reason why you skipped the parentheses here? are you trying to follow the style of the project that I cannot see in this PR? if not I'd say we stick to parens, which is what we use in the other ruby projects

File: lib/piwik/events.rb [hunk 2]
```diff
@@ -12,4 +12,40 @@ class Events < ApiModule
...
+  def action params = {}
+    getAction(defaults.merge(params))
+  end
+
+  def name params = {}
+    getName(defaults.merge(params))
+  end
+
+  def action_from_category params = {}
+    getActionFromCategoryId(defaults.merge(params))
```
Comment (serch) [inline lib/piwik/events.rb:29]: same questions goes for camelCase vs snake_case

**Review by serch — APPROVED** [review]

**Review by serch — COMMENTED** [review]
please add a description to the PR, no matter how small, something, why are we doing this, link to the issue maybe. if we ever come to this PR in the future would be nice to get a quick idea of what it does without having to dig through the code

File: lib/piwik/events.rb (hunk 1 above)
Comment (felipecabargas) [reply lib/piwik/events.rb:16]: Just using same style as rest of the project. We can add to the entire project in a separate PR.

File: lib/piwik/events.rb (hunk 2 above)
Comment (felipecabargas) [reply lib/piwik/events.rb:29]: `getActionFromCategory` is the method from Matomo's API. If you check the [whole file](https://github.com/Monsido/piwik-ruby-api/blob/8d04c7746b28cd5629994e1963fc3c258f1a2c3a/lib/piwik/events.rb) you can see the way the gem is structured.

**Pull request merged**